src/pandas_profiling/model/pandas/correlations_pandas.py

```diff
@@ -35,7 +40,68 @@ def pandas_pearson_compute(
 def pandas_kendall_compute(
     config: Settings, df: pd.DataFrame, summary: dict
 ) -> Optional[pd.DataFrame]:
-    return df.corr(method="kendall")
+
```

Comment:
The spark implementation should be placed in `src/pandas_profiling/model/spark/correlation_spark.py`